modules/creating-infra-machines.adoc

```diff
@@ -77,6 +83,13 @@ rendered-worker-daa08cc1e8f5fcdeba24de60cd955cc3            365c1cfd14de5b0e3b85
 
 . Optional: To deploy changes to a custom pool, create a machine config that uses the custom pool name as the label, such as `infra`. Note that this is not required and only shown for instructional purposes. In this manner, you can apply any custom configurations specific to only your infra nodes.
 +
+[NOTE]
+====
+Creating a machine config requires the associated nodes to reboot.
```

Comment:
To newbs, this might be a little difficult to digest.  I'm not an MCO expert, so the foll is a *complete* guess:

After you create a machine config, the MCO reboots the nodes that are associated with the machine config to apply the configuration.